wgpu-hal/src/dx12/adapter.rs

```diff
@@ -127,6 +127,11 @@ impl super::Adapter {
             )
         });
 
+        // If we don't have dxc, we reduce the max to 5.1
+        if dxc_container.is_none() {
+            shader_model_support.HighestShaderModel = d3d12_ty::D3D_SHADER_MODEL_5_1;
```

Comment:
good addition, reminder to myself to update #5154 to check `shader_model_support.HighestShaderModel >= d3d12_ty::D3D_SHADER_MODEL_6_0` instead of whether dxc is present for 64bit support